.github/workflows/container.yaml

```diff
@@ -0,0 +1,53 @@
+name: Create and publish a Docker image to ghcr on main and nightly with ko
+
+on:
+  push:
+    paths:
+      - "**.go"
+      - "**.yaml"
+  # Enable this when iterating on PRs
+  pull_request:
+    paths:
+      - "**.yaml"
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.platform }}
+      cancel-in-progress: true
+
+    strategy:
```

Comment:
There shouldn’t be a need for the matrix with ko. It supports multi arch already with one command.